contracts-v0.2/src/Aggregator.sol

```diff
@@ -128,20 +110,17 @@ contract Aggregator is Ownable, IAggregator, ITypeAndVersion {
 
     /**
      * @notice called by oracles when they have witnessed a need to update
-     * @param _roundId is the ID of the round this submission pertains to
      * @param _submission is the updated data that the oracle is submitting
      */
-    function submit(uint256 _roundId, int256 _submission) external {
-        bytes memory error = validateOracleRound(msg.sender, uint32(_roundId));
+    function submit(int256 _submission) external {
+        uint32 roundId = reportingRoundId + 1;
+        bytes memory error = validateOracleRound(msg.sender, uint32(roundId));
```

Comment:
There is no need to convert `roundId` to `uint32` anywhere in this function (every line of this function).